tests/sentinel/run.tcl

```diff
@@ -14,7 +14,7 @@ proc main {} {
         set ::env(LEAKED_FDS_FILE) $::leaked_fds_file
     }
     spawn_instance sentinel $::sentinel_base_port $::instances_count [list "sentinel deny-scripts-reconfig no"] "../tests/includes/sentinel.conf"
```

Comment:
maybe we should enable these protected features for the sentinel instances too?
let's do that, even if you don't have to.